storybook/src/server/lib/TwigComponentResolver.ts

```diff
@@ -3,24 +3,40 @@ import path from 'path';
 import dedent from 'ts-dedent';
 
 export class TwigComponentResolver {
-    constructor(private config: TwigComponentConfiguration) {}
+    constructor(
+        private config: TwigComponentConfiguration,
+        private templatePathAliases: {
+            [p: string]: string;
+        }
+    ) {}
 
     resolveNameFromFile(file: string) {
         const stripDirectory = (file: string, dir: string) => {
             return file.replace(dir, '').replace(/^\//, '').replaceAll('/', ':').replace('.html.twig', '');
         };
 
+        const resolvePathAlias = (file: string) => {
+            for (const [alias, resolvedPath] of Object.entries(this.templatePathAliases)) {
+                if (file.startsWith(alias)) {
+                    return file.replace(alias, resolvedPath);
+                }
+            }
+            return file;
+        }
+
+        const resolvedFile = resolvePathAlias(file);
```

Comment:
Replace host path with Docker path

This must be adapted for additional watch paths

> Ignoring additional watch path '/application/front/assets': path doesn't exists.

Reply:
Additional watch path can handle paths relative to `projectDir` or absolute without the need of `projectPathAliases`

Reply:
Not required anymore